### ISSUE TITLE
GET /v3/conversations returns 405 http code

### ISSUE THREAD

**Issue by apetrovs** [MicrosoftDocs/bot-docs#1357]
GET /v3/conversations returns 405 HTTP code but should return a list of conversations the bot has participated in (according to this https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#get-conversations) 

**Steps to reproduce the behavior**:
1. Get access token
2. Perform GET /v3/conversations
3. See the response

**Request**:
```
GET /amer/v3/conversations HTTP/1.1
Authorization: Bearer {access_token}
Host: smba.trafficmanager.net
Connection: close
User-Agent: Paw/3.1.8 (Macintosh; OS X/10.15.0) GCDHTTPRequest
```

**Response**:
```
HTTP/1.1 405 Method Not Allowed
Allow: POST
Content-Length: 72
Content-Type: application/json; charset=utf-8
Server: Microsoft-HTTPAPI/2.0
Date: Fri, 20 Sep 2019 12:31:51 GMT
Connection: close

{"message":"The requested resource does not support http method 'GET'."}
```

**Additional context**
This issue looks like #1264
I tried two endpoints: https://smba.trafficmanager.net/apis and https://smba.trafficmanager.net/amer.
Also, tried different networks and routings.
Clients: Paw and Postman. 

Feel free to ask any additional details.

**Comment by tdurnford** [comment]
@apetrovs What channel are you trying to use?

**Comment by tdurnford** [comment]
Closing due to inactivity.

**Comment by apetrovs** [comment]
@tdurnford Hey! Sorry for the late reply. What do you mean by "channel"?
I use https://smba.trafficmanager.net/amer as the service URL because I receive it in incoming events. And https://smba.trafficmanager.net/apis was mentioned in #1264, so I decided to try it also.

**Comment by tdurnford** [comment]
By channel I mean the BotFramework supported channels - Teams, Skype, Facebook Messenger, Line, Direct Line, etc. Basically how are you interacting with your bot?

**Comment by apetrovs** [comment]
@tdurnford thanks for the explanation. My bot is for Teams and I use the Java framework and REST API.

**Comment by tdurnford** [comment]
It appears that most channels do not support the get conversation endpoint which is why the request is responding with a 405 error. From what I can tell, Direct Line is the only channel that supports the request. This is not well documented unfortunately. Take a look at https://github.com/microsoft/BotFramework-Services/issues/88 for more details.

**Comment by dgancho** [comment]
@JonathanFingold I am using GET Conversations in Teams, and it returns "Method not allowed". You mentioned "Please comment on that thread, and we'll try to get you unblocked". What do I need to do to get this unblocked for my bots?

**Comment by apetrovs** [comment]
@tdurnford thanks for the information! But it would be great if that appears in documentation.

**Comment by JonathanFingold** [comment]
> 
> 
> @JonathanFingold I am using GET Conversations in Teams, and it returns "Method not allowed". You mentioned "Please comment on that thread, and we'll try to get you unblocked". What do I need to do to get this unblocked for my bots?

@tdurnford Can you also help @dgancho ? If it's more convenient, feel free to reopen the other issue I closed as a duplicate of this one.

**Comment by tdurnford** [comment]
@JonathanFingold It is the same issue. The get conversations get request does not support the Teams Channel. I don't believe there are any plans to change this behavior.

**Comment by penguinsource** [comment]
Can you please take a look at this related issue - https://github.com/MicrosoftDocs/bot-docs/issues/1934 ?

**Comment by erhopf** [comment]
Thanks for your dedication to our documentation. Unfortunately, at this time we have been unable to review your issue in a timely manner and we sincerely apologize for the delayed response. We are closing this issue for now, but if you feel that it's still a concern, please respond and let us know. If you determine another possible update to our documentation, please don't hesitate to reach out again. #please-close
